packages/neuron-ui/src/widgets/Select/index.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { useRef, useState, useEffect, useCallback } from 'react'
+import { useDidMount } from 'utils'
+import styles from './select.module.scss'
+
+export interface SelectOptions {
+  label: React.ReactNode
+  value: string
+  data?: any
+  className?: string
+}
+
+export interface SelectProps {
+  options: SelectOptions[]
+  className?: string
+  disabled?: boolean
+  onChange?: (arg: SelectOptions) => void
+  value?: SelectOptions | string
+  placeholder?: String
+}
+
+function parseValue(value: string | SelectOptions, options: SelectOptions[]) {
+  let option: SelectOptions | undefined
+
+  if (typeof value === 'string') {
+    for (let i = 0, num = options.length; i < num; i++) {
+      if (typeof options[i].value !== 'undefined' && options[i].value === value) {
```

Comment:
`typeof options[i].value !== 'undefined'` is redundant since `options[i].value === value` covers it. and `Array#find` can be used here.